### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by mleimeister**
Work in progress for adding German translations to TWC Berlin site. Opening this PR to share the preview with others for feedback.

### CODE REVIEW

**Comment by shushugah** [comment]
So far this looks great and ready to ship! I noticed in a few places, you used `/events` instead of `/de/events` and one benefit
Screenshot 2020-07-20 at 23 58 59
 I'd say for using `/de/events` it helps analytics understand how many people would benefit from German events, and also the date/times are translated into German, as shown in this screenshot

File: _i18n/en.yml [hunk 1]
```diff
@@ -31,7 +32,7 @@ connect:
 global:
   code_of_conduct:
     title: Code of Conduct
-    summary: All meetings and communications are covered under the a <a rel="noopener noreferrer" target="_blank" href="https://berlincodeofconduct.org">Berlin Code of Conduct</a>. The privacy and safety of our members is important to us. Sharing the identity of members and or taking/sharing photos are strictly forbidden unless express consent is given.
```
Comment (shushugah) [inline _i18n/en.yml:34]: nice!

**Review by shushugah — APPROVED** [review]
I cannot comment on German quality, but this looks great!

**Comment by mleimeister** [comment]
> I cannot comment on German quality, but this looks great!

Great, thanks! I'd show it to one or two other German speakers to get feedback on the tone, then can finalize it.

**Pull request merged**